ospfd/ospf_vty.c

```diff
@@ -6368,9 +6818,12 @@ DEFUN (no_ip_ospf_dead_interval,
   /* Update timer values in neighbor structure. */
   if (argc == 1)
     {
-      struct ospf *ospf;
-      
-      if ((ospf = ospf_lookup()))
+      struct ospf *ospf = ospf_lookup_by_vrf_id (ifp->vrf_id);
+
+      if (IS_DEBUG_OSPF_TRACE)
```

Comment:
I'm going to stop looking at every new debug in this file, but if they must provide a person coming through later the ability to figure out what is going on else they are useless.  What are we providing that is useful long term?